ipaplatform/redhat/authconfig.py

```diff
@@ -85,10 +87,16 @@ def execute(self, update=True):
             self.add_option("update")
 
         args = self.build_args()
-        ipautil.run(["/usr/sbin/authconfig"] + args)
+        try:
+            ipautil.run([paths.AUTHCONFIG] + args)
+        except ipautil.CalledProcessError:
+            print("Failed to execute authconfig command")
 
     def backup(self, path):
-        ipautil.run(["/usr/sbin/authconfig", "--savebackup", path])
+        try:
+            ipautil.run([paths.AUTHCONFIG, "--savebackup", path])
+        except ipautil.CalledProcessError:
+            print("Failed to execute authconfig command")
```

Comment:
I'd prefer to see the error message returned from the process instead of a generic one.

It seem that `ipautil.run` can return this error message when `capture_errors=True`. See https://github.com/freeipa/freeipa/blob/master/ipapython/ipautil.py#L322 for the relevant code.